src/helpers/columnWidth/columnWidth.js

```diff
@@ -1,13 +1,18 @@
 import { css } from 'styled-components';
 import has from 'lodash/has';
+import isUndefined from 'lodash/isUndefined';
 import isString from 'lodash/isString';
 
 import { themeProvider } from '../../theme';
 
 const { theme } = themeProvider;
 
-export const percentage = (props, breakpoint) =>
-  (Math.abs(props[breakpoint]) / theme(props).columns) * 100;
+export const percentage = (props, breakpoint) => {
+  const value = has(props, `${breakpoint}`)
+    ? props[breakpoint]
+    : theme(props).col[breakpoint];
+  return (Math.abs(value) / theme(props).columns) * 100;
```

Comment:
If you return undefined as a value, does this line work fine or does it throw an error?

Reply:
It doesn't throw an error but you are right, there is no reason to have code smells. Just committed a fix https://github.com/aaronvanston/react-flexa/pull/78/commits/a306b725f71e918cf41e4b50f8763c1062c386a1

Reply:
Thanks @sstefoss! Will do a final test through and merge soon! :)